src/platform/silabs/platformAbstraction/SilabsPlatform.h

```diff
@@ -57,6 +57,12 @@ class SilabsPlatform : virtual public SilabsPlatformAbstractionBase
 
     void StartScheduler(void) override;
 
+#ifdef SL_MATTER_RGB_LED_ENABLED
```

Comment:
revisit the #ifdef SL_MATTER_RGB_LED_ENABLED based on my previous comment

I will not copy for all the other but please apply to all instances

Reply:
Addressed by adding `#if (defined(SL_MATTER_RGB_LED_ENABLED) && SL_MATTER_RGB_LED_ENABLED == 1)`